app/views/task_list/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<!-- TaskList#index -->
+
+<!-- <h1 class="index-heading">Task List</h1> -->
+<nav class="menu-container">
```

Comment:
I suggest this `nav` should go in the layout file so it can be consistent between pages.